profile/README.md

```diff
@@ -1 +1,46 @@
 # Massa
+
+Welcome aboard on the *Massa* organisation. On this profile you will find every repositories related to the development on the **Massa blockchain**.
+
+## Code
+
+- [massa](https://github.com/massalabs/massa): Main code of the Massa blockchain, builds the node and client, holds all the critical code for the architecture.
+- [massa-proto](https://github.com/massalabs/massa-proto): Repository centralizing protobuf bindings for Massa GRPC.
+- [massa-bridge](https://github.com/massalabs/massa-bridge): Blockchain bridge between Massa and any EVM-compatible blockchains.
+- [PeerNet](https://github.com/massalabs/PeerNet): P2P network backend for the Massa blockchain.
+- [stream_limiter](https://github.com/massalabs/stream_limiter): Rust library to implement read/write speed limits on any stream.
+
+## Smart contracts
+
+- [massa-sc-runtime](https://github.com/massalabs/massa-sc-runtime): Sandbox to run the smart-contracts (compiled as WASM) on a Massa node.
+- [massa-as-sdk](https://github.com/massalabs/massa-as-sdk): SDK to build smart-contracts in AssemblyScript. Abstracts the Massa's ABI function.
+- [massa-sc-toolkit](https://github.com/massalabs/massa-sc-toolkit): Provide tools such as `@massalabs/sc-project-initializer` to improve the experience of smart-contracts development.
+- [massa-sc-compiler](https://github.com/massalabs/massa-sc-compiler): Compile a smart-contract to a WASM binary.
+- [massa-sc-playground](https://github.com/massalabs/massa-sc-playground): Playground in AssemblyScript for smart-contracts.
```

Comment:
@Ben-Rey @AurelienFT not sure that this repo needs to be mentioned. If there was no developments after October on this topic, I suggest we deprecate the repo and remove SC-playground from Documentation

Reply:
I'm personally ok with that